src/interfaces/agm-ext-eval-interface.ts

```diff
@@ -1,5 +1,6 @@
 /** Object that stores AGM Extension evaluation fields. */
 export interface AgmExtEvalIF {
+  isEligible: boolean
```

Comment:
When I merge my PR #571 , you'll see I added this to the end of the props list. Doesn't matter either way.

I also initialized it to False instead of null below, meaning no-one has to update it for it to be correct initially. This doesn't really matter since it will probably be set initially by Karim's component logic.

Reply:
571 is merged. Just watch out that the merge doesn't add your prop in a second time.